app/recordtransfer/apps.py

```diff
@@ -238,27 +245,112 @@ def verify_accepted_file_formats() -> None:
             inverted_formats[extension] = group_name
 
 
-def verify_caais_defaults():
-    """Verifies the setting:
+def verify_caais_defaults() -> None:
+    """Verify the settings.
 
     - CAAIS_DEFAULT_SUBMISSION_EVENT_TYPE
     - CAAIS_DEFAULT_CREATION_TYPE
 
-    Ensures that all required defaults exist, raises an ImproperlyConfigured
-    exception if a default does not exist.
+    Raises:
+        ImproperlyConfigured: If either setting is empty.
     """
     if not settings.CAAIS_DEFAULT_SUBMISSION_EVENT_TYPE:
         raise ImproperlyConfigured("CAAIS_DEFAULT_SUBMISSION_EVENT_TYPE is not set")
     if not settings.CAAIS_DEFAULT_CREATION_TYPE:
         raise ImproperlyConfigured("CAAIS_DEFAULT_CREATION_TYPE is not set")
 
 
+def _validate_cron(cron_str: str) -> None:
+    """Validate a cron expression.
+
+    Args:
+        cron_str: The cron expression to validate.
+
+    Raises:
+        ValueError: If the cron expression is invalid.
+    """
+    # Check number of fields
+    fields = cron_str.strip().split()
+    if len(fields) != 5:
+        raise ValueError("Cron expression must have 5 fields")
+
+    # Regex for each cron field
+    cron_patterns = {
+        "minute": r"^(?:\*(?:/[1-9]\d*)?|[0-5]?\d(?:-[0-5]?\d)?(?:/[1-9]\d*)?(?:,[0-5]?\d)*)$",
+        "hour": r"^(?:\*(?:/[1-9]\d*)?|[01]?\d|2[0-3](?:-[01]?\d|2[0-3])?(?:/[1-9]\d*)?(?:,[01]?\d|2[0-3])*)$",
+        "day_of_month": r"^(?:\*(?:/[1-9]\d*)?|[1-9]|[12]\d|3[01](?:-[1-9]|[12]\d|3[01])?(?:/[1-9]\d*)?(?:,[1-9]|[12]\d|3[01])*)$",
+        "month": r"^(?:\*(?:/[1-9]\d*)?|[1-9]|1[0-2](?:-[1-9]|1[0-2])?(?:/[1-9]\d*)?(?:,[1-9]|1[0-2])*)$",
+        "day_of_week": r"^(?:\*(?:/[1-9]\d*)?|[0-6](?:-[0-6])?(?:/[1-9]\d*)?(?:,[0-6])*)$",
+    }
+
+    field_names = ["minute", "hour", "day_of_month", "month", "day_of_week"]
+
+    # Validate each field
+    for value, field in zip(fields, field_names):
+        pattern = cron_patterns[field]
+        if not re.match(pattern, value):
+            raise ValueError(f"Invalid value '{value}' for field '{field}'")
+
+
+def verify_upload_session_expiry_settings() -> None:
+    """Verify the settings.
+
+    - UPLOAD_SESSION_EXPIRED_CLEANUP_SCHEDULE
+    - UPLOAD_SESSION_EXPIRE_AFTER_INACTIVE_MINUTES
+    - UPLOAD_SESSION_EXPIRING_REMINDER_MINUTES
+
+    Raises:
+        ImproperlyConfigured: If the expiry schedule is not a valid cron string, or if the
+        expiry settings are invalid.
+    """
+    try:
+        _validate_cron(settings.UPLOAD_SESSION_EXPIRED_CLEANUP_SCHEDULE)
```

Comment:
This setting should be allowed to be empty, since an empty string will disable this feature.

```suggestion
        cron_str = settings.UPLOAD_SESSION_EXPIRED_CLEANUP_SCHEDULE
        if cron_str:
            _validate_cron(cron_str)
```